com_migratetojoomla/admin/src/controller/DisplayController.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
The controller parent folder should be 'Controller', not 'controller'.

Reply:
Still not changed apparently :-)

Reply:
In my local name is Controller don't know why here lowercase